repos/system_upgrade/common/libraries/rpms.py

```diff
@@ -114,3 +152,42 @@ def check_file_modification(config):
     """
     output = _read_rpm_modifications(config)
     return _parse_config_modification(output, config)
+
+
+def get_leapp_packages(major_version=None, component=None, include_deps=False):
+    """
+    Get list of leapp packages.
+
+    :param major_version: a list or string specifying major_versions. If not defined then current
+        system_version will be used.
+    :param component: a list of or a single enum value specifying leapp components
+        (use enum :class: LeappComponents) If defined then only packages related to the specific
+        component(s) will be returned.
```

Comment:
SUGGESTION_TAG: easier_component_modification

```suggestion
        (use enum :class: LeappComponents) If defined then only packages related to the specific
        component(s) will be returned. The default set of components is in
        `LEAPP_DEFAULT_COMPONENTS` and simple modifications of the default can be achieved
        with code like:
        .. code-block:: python
            get_leapp_packages(
                component=LEAPP_DEFAULT_COMPONENTS.difference(
                    [LeappComponents.TOOLS]
            ))
```

Reply:
This (and the code to go along with it) has been added as https://github.com/oamg/leapp-repository/pull/1156/commits/e61b212aa845af70c30439498629fed74eb1be11

@pirat89 @fernflower If you like the change in that commit, we can squash everything together.  If you don't like it, we can revert just that one commit and discuss it in a different PR.